zulipterminal/ui_tools/views.py

```diff
@@ -179,6 +179,15 @@ def update_search_box_narrow(self, message_view: Any) -> None:
 
     def read_message(self) -> None:
         # Message currently in focus
+        if hasattr(self.model.controller, "view"):
+            self.view = self.model.controller.view
+        else:
+            return
```

Comment:
Can this just use a local variable for view? I'm not sure why you're setting it on the object rather than just a local variable in the method.

Since `self.view` is not used later at all, it could also be written with `getattr` with a default, or with an exception block?

Reply:
I might have done it out of habit. Thanks for pointing out.